src/regolith/exemplars.json

```diff
@@ -2169,6 +2169,49 @@
             "year": 2020
         }
     ],
+    "releaseReports": [
+        {
+            "_id": "sbillinge",
+            "date": "17/07/2025",
```

Comment:
dates are in iso standard.  Also, what date is this?, when the package was first created?  in which case `creation_date` or something like that.